cmd/doctor.go

```diff
@@ -22,18 +25,24 @@ import (
 // CmdDoctor represents the available doctor sub-command.
 var CmdDoctor = cli.Command{
 	Name:        "doctor",
-	Usage:       "Diagnose the problems",
-	Description: "A command to diagnose the problems of current gitea instance according the given configuration.",
+	Usage:       "Diagnose and repair problems",
+	Description: "A command to diagnose and repair problems of current gitea instance according the given configuration.",
 	Action:      runDoctor,
 }
 
 type check struct {
 	title string
 	f     func(ctx *cli.Context) ([]string, error)
+	abort bool
```

Comment:
Can this variable be renamed to indicate it is to abort if there are errors with a particular check?  
`abortIfError` doesn't look very pretty but makes more sense to me.

Reply:
> 
> 
> Can this variable be renamed to indicate it is to abort if there are errors with a particular check?
> `abortIfError` doesn't look very pretty but makes more sense to me.

Done.